definitions/divorce/json/AuthorisationCaseEvent/AuthorisationCaseEvent-nonprod.json

```diff
@@ -54,5 +54,12 @@
     "CaseEventID": "tempEditDerivedAddresses",
     "UserRole": "caseworker-divorce-superuser",
     "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "29/07/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "amendPetitionForRefusalRejection",
+    "UserRole": "[PETSOLICITOR]",
+    "CRUD": "CRU"
```

Comment:
I think you can also add "R" permission to the following roles:
- caseworker-divorce-courtadmin_beta
- caseworker-divorce-superuser
- caseworker-divorce-courtadmin-la
This is a convention we've had for events, to give CRU to the role that needs to use the event, and "R" for the other roles. I can't remember exactly why, I think it was related that it helps if other roles have at least "R" role when doing statistics related type of work.

Reply:
Thanks @Dizrupto. Had the same questions in the [Jira ticket](https://tools.hmcts.net/jira/browse/DIV-6206?focusedCommentId=795811&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-795811). I'll take this as your response.

Reply:
Updated.